Kas 2561 pub from mr  decision date

mdv-professional opened the pull request:
[Jira](https://kanselarij.atlassian.net/browse/KAS-2561)

zie ook [frontend](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/930)

Commit:
KAS 2561 main | decision date [pub from mr]

Commit:
KAS 2561 rename start date | decision date [pub from mr]

Commit:
KAS 2561 migration | decision date [pub from mr]

Commit:
KAS 2561 migration fix | decision date [pub from mr]

ValenberghsSven committed:
Merge branch 'development' of https://github.com/kanselarij-vlaanderen/kaleidos-project into KAS-2561-pub-from-mr--decision-date

ValenberghsSven commented on config/resources/publicatie-domain.json:
```diff
@@ -114,6 +114,12 @@
           "predicate": "adms:identifier",
           "target": "identification",
           "cardinality": "one"
+        },
+        "agenda-item-treatment": {
+          "predicate": "dct:subject",
+          "inverse": true,
+          "cardinality": "one",
+          "target": "agenda-item-treatment"
```
moet de relatie hier niet omgekeerd ?
![image](https://user-images.githubusercontent.com/22245223/121364188-da6ea280-c937-11eb-9fdb-0701351722a2.png)

Publicatie heeft als onderwerp de beslissing.

erikap committed:
Merge branch 'development' into KAS-2561-pub-from-mr--decision-date

erikap committed:
Fix direction of relation between publication and treatment

erikap committed:
Fix migration to add startdate to all existing treatments

- Apply change in all graphs, not only kanselarij-graph
- Add datatype to the inserted date
- Use correct format YYYY-MM-DD for the date string

erikap committed:
Merge branch 'development' into KAS-2561-pub-from-mr--decision-date

erikap committed:
Add write access on agendaitem treatment for OVRB users
